app/src/main/java/com/example/lamnn/todo_mvp/data/model/Task.java

```diff
@@ -0,0 +1,59 @@
+package com.example.lamnn.todo_mvp.data.model;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.Serializable;
+
+public class Task implements Serializable {
+  private int mUserId;
+  private int mId;
+  private String mTitle;
+  private boolean mCompleted;
+
+  public Task(JSONObject jsonObject) throws JSONException {
+    mUserId = jsonObject.getInt(JSONKey.USER_ID);
+    mId = jsonObject.getInt(JSONKey.ID);
+    mTitle = jsonObject.getString(JSONKey.TITLE);
+    mCompleted = jsonObject.getBoolean(JSONKey.COMPLETED);
+  }
+
+  public int getmUserId() {
```

Comment:
em làm theo tip trick số 6
https://viblo.asia/p/mot-so-tips-tricks-khi-lam-viec-voi-android-studio-yMnKMJjgZ7P
sau đó xóa getter setter đi gen lại.
Sẽ được kết quả như mong đợi nhé
```suggestion
  public int getUserId() {
```